### ISSUE TITLE
Install URL does not work with ARM64 hosts

### ISSUE THREAD

**Issue by carlosedp** [coursier/coursier#1988]
The URL `https://git.io/coursier-cli-"$(uname | tr LD ld)"` currently doesn't work with ARM64 hosts like ARM64 Linux.

To override this I currently use:

```sh
curl -OL $(curl -s "https://api.github.com/repos/coursier/coursier/releases/latest" | grep "$(uname | tr LD ld)" |grep "$(uname -m)" | grep "browser_download_url" | cut -d '"' -f 4)
```

Maybe an option would be having the URLs like:

`https://git.io/coursier-cli-"$(uname -m)"-"$(uname | tr LD ld)"` and name the releases as such (removing the pc or mac).

**Comment by scalway** [comment]
I guess this link is always walid:
```
https://raw.githubusercontent.com/coursier/launchers/master/cs-aarch64-pc-linux
```
and it already exist on:
```
https://git.io/coursier-cli-linux-arm64
```
And yes ... I've made it. It shows that ANY script that creates link is an abuse! Even in simple case like above if `uname  | tr LD ld` returns something unexpected it will create link that anyone can create right now to make infected program!

**Comment by juhp** [comment]
Yes, +1 to all of that.

I had to manually determine the link for aarch64 Linux.

I want to suggest adding an alternative direct link to the GitHub binary directory.

(While putting the artefacts directly in git gives history, I don't think it is considered best practice: creating them as GitHub release outputs would be best if possible.)

**Comment by mkurz** [comment]
Willl be fixed by #2640 ?
